### ISSUE TITLE
Stable version

### ISSUE THREAD

**Issue by ayrat555** [rusterlium/rustler#364]
When are you planning to release a stable version of the project? Currently, it's not possible to release stable versions of libraries that depend on pre-release version of the rustler.

```
> mix hex.publish
** (Mix) A stable package release cannot have a pre-release dependency
```

**Comment by evnu** [comment]
We are getting there. I just released a rc.2 to test the fixes introduced since rc.1. I think we will release 0.22.0 within this week. (Sorry for taking so long for this)

**Comment by evnu** [comment]
We just released 0.22.0.
